spec/hello_spec.rb

```diff
@@ -1,13 +1,14 @@
 require "spec_helper"
 
-describe "#hello_t" do 
+describe "#hello_t" do
   let (:names) { ["Tim", "Tom", "Jim"] }
-  it "only passes names that start with 'T' to the block" do 
+
+  it "only passes names that start with 'T' to the block" do
     expect{hello_t(names){|name| puts "Hi, #{name}" if name.start_with?("T") }}.to output("Hi, Tim\nHi, Tom\n").to_stdout
```

Comment:
I really don't like the {} syntax with any sort of "logic". Soooo much is happening on this one line of code! Can we separate this out into a few lines of code so that's it's more straightforward to read?